src/Swig/pxr/usd/usdRi/tokens_Tokens.i

```diff
@@ -39,11 +39,17 @@
   public static TfToken aovName = new TfToken("aovName");
```

Comment:
That's a side question, maybe more for @jcowles ... I'm wondering why we have any work/bind for the usdRi API as we don't use RenderMan at all?